app/src/main/java/org/mozilla/fenix/components/metrics/AppAllSourceStartTelemetry.kt

```diff
@@ -1,59 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-package org.mozilla.fenix.components.metrics
```

Comment:
Ah sweet, I didn't realize git move existed.. thanks.

In this case, I didn't really rename anything. I started with a fresh file and wrote everything new, it just looks similar because I used the same concept I guess 😅